app/config/question-bank.js

```diff
@@ -3118,24 +3143,36 @@ const questionBank = {
               type: 'input',
               classes: 'govuk-input--width-20',
               label: {
-                html: 'Building and street <span class="govuk-visually-hidden">line 1 of 2</span>',
+                text: 'Address line 1',
                 classes: 'govuk-label'
               },
               validate: [
                 {
                   type: 'NOT_EMPTY',
-                  error: 'Enter your building and street details'
-                }
+                  error: 'Enter your address line 1'
+                },
+                {
+                  type: 'REGEX',
+                  regex: ADDRESS_REGEX,
+                  error: 'Address must only include letters, numbers, hyphens and apostrophes'
+                },
               ]
             },
             {
               yarKey: 'address2',
               type: 'input',
               classes: 'govuk-input--width-20',
               label: {
-                html: '<span class="govuk-visually-hidden">Building and street line 2 of 2</span>',
+                html: 'Address line 2 (optional)',
                 classes: 'govuk-label'
-              }
+              },
+              validate: [
+                {
+                  type: 'REGEX',
+                  regex: ADDRESS_REGEX,
+                  error: 'Address must only include letters, numbers, hyphens and apostrophes'
```

Comment:
the ADDRESS regex also includes commas and full stops, are these also allowed in the Address field/s?